boards/arm/b_u585i_iot02a/b_u585i_iot02a-common.dtsi

```diff
@@ -54,6 +54,21 @@
 		watchdog0 = &iwdg;
 		spi-flash0 = &mx25lm51245;
 	};
+
+	fstab {
```

Comment:
Other boards have the fstab node defined by a overlay file in the  samples/subsys/fs/littlefs
Why adding it in the DTS of the board just for b_u585i_iot02a and stm32l562e_dk ?

Reply:
This is more about enable those STM boards than perform refactor. I'm fine to define fstab as an overlay. Let me know how to proceed.